models/pp_cond_exp_mc_urbanczik.h

```diff
@@ -107,82 +107,140 @@ class pp_cond_exp_mc_urbanczik_parameters
   double tau_syn_in[ NCOMP ]; //!< Rise time of inhibitory synaptic conductance in ms
 };
 
+/* BeginUserDocs: neuron, point process, conductance-based
 
-/** @BeginDocumentation
-@ingroup Neurons
-@ingroup pp
-@ingroup cond
-
+Short description
++++++++++++++++++
 
-Name: pp_cond_exp_mc_urbanczik - Two-compartment point process neuron with conductance-based synapses
+Two-compartment point process neuron with conductance-based synapses
 
-Description:
+Description
++++++++++++
 
 pp_cond_exp_mc_urbanczik is an implementation of a two-compartment spiking
 point process neuron with conductance-based synapses as it is used
-in [1]. It is capable of connecting to a Urbanczik synapse.
+in [1]_. It is capable of connecting to an :doc:`Urbanczik synapse
+<urbanczik_synapse>`.
 
-The model has two compartments: soma and dendrite, labeled as s and p, respectively.
-There is one excitatory and one inhibitory conductance-based synapse onto each compartment.
-Each compartment can also receive current input from a current generator, and an external
-(rheobase) current can be set for each compartment.
+The model has two compartments: soma and dendrite, labeled as s and p,
+respectively. There is one excitatory and one inhibitory conductance-based
+synapse onto each compartment. Each compartment can also receive current input
+from a current generator, and an external (rheobase) current can be set for
+each compartment.
 
 Synapses, including those for injection external currents, are addressed through
 the receptor types given in the receptor_types entry of the state dictionary.
 Note that in contrast to the single-compartment models, all
 synaptic weights must be positive numbers!
 
-
-Parameters:
+.. _multicompartment-models:
+
+Multicompartment models and synaptic delays
++++++++++++++++++++++++++++++++++++++++++++
+
+Note that in case of multicompartment models which represent the dendrite
+explicitly, the interpretation of the synaptic delay in NEST requires a careful
+consideration. In NEST, the delay is at least one simulation time step and is
+assumed to be located entirely at the postsynaptic side. For point neurons, it
+represents the time it takes for an incoming spike to travel along the
+postsynaptic dendrite before it reaches the soma, see :ref:`panel a)
+<fig-multicompartment>`. Conversely, if the synaptic weight depends on the
+state of the postsynaptic neuron, the delay also represents the time the
+information on the state propagates back through the dendrite to the synapse.
+
```

Comment:
"For multicompartment models, this amounts to positioning the delay directly behind the incoming synapse. . ."
@jstapmanns Is the 'positioning of the delay' something the user does when setting up a simulation or is it inherent in the model code itself? As in is this a variable that can be modified by a user in a Python script? If so, I was wondering if along with the figure, it would be a good idea to show a code snippet of a script. So you can see how the biological side is represented in NEST.

Reply:
No, the position of the delay is not configurable by the user. It is a basic property of NEST that it is always located on the postsynaptic side.

Reply:
@jstapmanns Ok, thanks for the clarification; I think it might be helpful reword it (in my reading of it it's not clear who is doing  the 'positioning')
Does this make sense "For multicompartment models in NEST, this means the delay is positioned directly behind the incoming synapse . . ."

Reply:
@jessica-mitchell I agree with your suggestion.